<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:rgb(234, 234, 234);}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>630</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;background-color:rgb(225, 134, 127);
font: 24pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>Please enter the required details below.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_edit_phone_number">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>450</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(176, 176, 176);
font: 20pt &quot;Calibri&quot;;color:rgb(79, 79, 79)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_edit_ic_number">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>550</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(176, 176, 176);
font: 20pt &quot;Calibri&quot;;color:rgb(79, 79, 79)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_phone_number">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>Phone number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ic_number">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>IC number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>450</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Calibri&quot;;color:red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_edit_age">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(176, 176, 176);
font: 20pt &quot;Calibri&quot;;color:rgb(79, 79, 79)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_age">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_edit_name">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>721</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(176, 176, 176);
font: 20pt &quot;Calibri&quot;;color:rgb(79, 79, 79)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_edit_address">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>440</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(176, 176, 176);
font: 20pt &quot;Calibri&quot;;color:rgb(79, 79, 79)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>410</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_edit_state">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>360</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(176, 176, 176);
font: 20pt &quot;Calibri&quot;;color:rgb(79, 79, 79)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_state">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>330</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>State</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goToSignIn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>740</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Sign in instead</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>630</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;background-color:rgb(225, 134, 127);
font: 24pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
